routing/localchans/manager.go

```diff
@@ -155,7 +172,58 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 					"not yet confirmed",
 				))
 
+		case createMissingEdge:
+			// If the edge was not found, but the channel is found,
+			// that means the edge is missing in the graph database
+			// and should be recreated. The edge and policy are
```

Comment:
Nit:

// The ~~edge and~~ policy are
// created in-memory and will be added to the graph in
// the PropagateChanPolicyUpdate call below.

Reply:
Updated to 
```
// If the edge was not found, but the channel is found,
// that means the edge is missing in the graph database
// and should be recreated. The edge and policy are
// created in-memory. The edge is inserted in createEdge
// below and the policy will be added to the graph in
// the PropagateChanPolicyUpdate call below.
```